docs/api/cases/cases-api-add-comment.asciidoc

```diff
@@ -0,0 +1,164 @@
+[[cases-api-add-comment]]
+== Add comment to case API
+++++
+<titleabbrev>Add comment</titleabbrev>
+++++
+
+Adds a comment to a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases/<case_id>/comments`
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases/<case_id>/comments`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the case you're updating.
+
+
+=== Path parameters
+
+`<case_id>`::
+(Required,string) The identifier for the case. To retrieve case IDs, use
+<<cases-api-find-cases>>.
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Request body
```

Comment:
@nastasha-solomon pointed me out that the `owner` is a required field that is not listed here. That is correct. I forgot to mention it.

Reply:
Fixed in 6ee0090bc62f6d1c70082cf625ba9a22fbabf064